pkg/kubelet/cm/devicemanager/types.go

```diff
@@ -81,6 +81,10 @@ type Manager interface {
 
 	// UpdateAllocatedDevices frees any Devices that are bound to terminated pods.
 	UpdateAllocatedDevices()
+
+	// GetExclusiveResources returns a list of resource names whose instances were esclusively
+	// allocated to this container.
+	GetExclusiveResources(pod *v1.Pod, container *v1.Container) []string
```

Comment:
For symmetry with the other functions in this interface, should this be called `GetAllocatedDevices()`? Also, how is this different than `GetDevices()` in terms of the underlying information that it provides?

Reply:
addressed below